tee-worker/client-api/parachain-api/prepare-build/interfaces/trusted_operations/definitions.ts

```diff
@@ -26,7 +26,7 @@ export default {
                 __Unused_evm_account_codes: "Null",
                 __Unused_evm_account_storages: "Null",
                 id_graph: "(LitentryIdentity)",
-                id_graph_stats: "(LitentryIdentity)",
+                __Unused_id_graph_stats: "(LitentryIdentity)",
```

Comment:
This is not strictly necessary though as it's the last item of the enum (thus nothing there in the first place

Reply:
If we assign this index to new getter it may cause problem for clients using old sdk in the future, that's the only risk a see but at this point I also opt for not having this.

Reply:
I'll remove it then 👍🏼